src/sgd/frontend/yeastgenome/__init__.py

```diff
@@ -10,6 +10,7 @@
 import sys
 import random
 import re
+import yaml
```

Comment:
Why do we need this library? You can parse json with `json.loads`

Reply:
this is to convert a string to a dictionary.